src/swift/codeGeneration.js

```diff
@@ -376,6 +381,15 @@ export function structDeclarationForSelectionSet(
   });
 }
 
+function operationIdentifier(generator,  { operationName, sourceWithFragments, operationId }) {
+  if (!generator.context.generateOperationIds) {
+    return
+  }
+
+  generator.printNewlineIfNeeded();
+  generator.printOnNewline(`public static let operationId = "${operationId}"`);
```

Comment:
Hm, yeah, you're right. Let's go with `operationIdentifier` instead (I'm looking at `URLSessionTask.taskIdentifier` as inspiration).